src/resmom_win/start_exec.c

```diff
@@ -1997,8 +1998,11 @@ finish_exec(job *pjob)
 			/* Need to launch a shell*/
 			launch_shell = 1;
 		}
+		/* If we fail to get cmd shell(unlikely), use "cmd.exe" as shell */
+		if (0 != get_cmd_shell(cmd_shell, sizeof(cmd_shell)))
```

Comment:
That's actually a very good reason to place the constant first. Unfortunately, it's not consistent with the rest of the code. I'm not going to be a stickler about it if you want to leave it as-is.